### PR TITLE
Fixed broken tests of angular 9 (WEB)

### PR DESCRIPTION
**Pull request by tayronemachadozup**
Este pr corrige os seguintes testes:

ButtonScreen.feature
SendRequestScreen.feature
SimpleFormScreen.feature
TextInputScreen.feature
ConfirmScreen.feature

Closes: #50

### CODE REVIEW

File: automated-tests/common/tsconfig.json [hunk 1]
```diff
@@ -2,9 +2,10 @@
   "compilerOptions": {
     "target": "es2015",
     "lib": ["es2015", "dom"],
-    "types": ["cypress"]
+    "types": ["cypress","@types/node"]
```
Comment (arthurbleilzup) [inline automated-tests/common/tsconfig.json:5]:
```suggestion
    "types": ["cypress", "node"]
```

File: automated-tests/common/cypress/support/page-objects/ConfirmPage.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ class ConfirmPage extends BeaglePage {
         return super.init().then(() => {
             cy.on('window:alert', (message) => this.lastAlertMessage = message)
             cy.on('window:confirm', (message) => this.lastConfirmMessage = message)
+            cy.wait(1000)
```
Comment (arthurbleilzup) [inline automated-tests/common/cypress/support/page-objects/ConfirmPage.ts:32]: Why should you wait?

**Review by arthurbleilzup — CHANGES_REQUESTED** [review]

File: automated-tests/common/cypress/support/page-objects/ConfirmPage.ts (hunk 2 above)
Comment (tayronemachadozup) [reply automated-tests/common/cypress/support/page-objects/ConfirmPage.ts:32]: I checked and it really is unnecessary. Thank you!

**Review by arthurbleilzup — APPROVED** [review]

**Review by arthurbleilzup — DISMISSED** [review]

**Review by arthurbleilzup — APPROVED** [review]

File: automated-tests/common/cypress/integration/TouchableScreen.feature [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        When I click on touchable text <touchable>
+        Then component should render the action attribute correctly
+    
+  Scenario Outline: Touchable 03 - Touchable component performs action click on image correctly
+        And I have an image with <touchable> configured
+        When I click on touchable image <touchable>
+        Then component should render the action attribute correctly
+    
+
+
```
Comment (igorgiumellizup) [inline automated-tests/common/cypress/integration/TouchableScreen.feature:41]: Too much spaces here

**Review by igorgiumellizup — APPROVED** [review]

**Review by igorgiumellizup — APPROVED** [review]

**Review by arthurbleilzup — APPROVED** [review]

**Review by Tiagoperes — APPROVED** [review]

**Pull request merged**